pootle/apps/pootle_data/utils.py

```diff
@@ -423,7 +423,11 @@ def aggregate_children(self, stats):
         latest = dict(last_submission=None, last_created_unit=None)
         last_submission_pk = None
         last_created_unit_time = None
-        for child in stats["children"].values():
+        for path, child in stats["children"].items():
+            if path.startswith("templates-") or path == "templates":
+                if not self.context.pootle_path.startswith("/templates/"):
```

Comment:
not really because its not always a template - it can be a directory or a tp etc